views/index.erb

```diff
@@ -1,59 +1,198 @@
+            
+            <h2 %=@profile.keys[1]%></h2>Date of birth: 
+            <h3 class="profile-value"><%=@profile["dob"]%></h3>
+
+            <h2><%=@profile.keys[4].capitalize%>:</h2> 
+            <h3 class="profile-value"><%=@profile["gender"]%></h3>    
+
+            <h2 %=@profile.keys[2]%></h2>Employment status: 
+            <h3 class="profile-value"><%=@profile["employment_status"]%></h3><br>
+  </div>
+  <h3>Interests</h3>
+  <table>
+    <tbody>
+    <tr>
+        <td><img src="/images/icon_travelling.png" /></td>   
+        <td><img src="/images/icon_coffee.png" /></td>       
+        <td><img src="/images/icon_music.png" /></td> 
+        <td><img src="/images/icon_horses.png" /></td> 
+        <td><img src="/images/icon_gym.png" /></td> 
+        <td><img src="/images/icon_draughts.png" /><td>     
+    </tr>
+    </tbody>
+  </table>
+  </div>
+</div>       
+    <!--Add a field to the form to add Codecademy account to the profile -->
+<div class="right">
+       
+    <div class="profile-section box">
+      <h3>Track Completedness</h3>
+       
+        <h3 class="profile-value">from Codecademy profile with the name: <a href="https://www.codecademy.com/IrynaBi">
+<p><%=@profile["codecademy"]%></a></h3><br>
+    
+    <div class="percentage"><p><h2 %=@profile.keys[7]%></h2></p><a href="https://www.codecademy.com/learn/javascript"> JavaScript Track</a></div>     
+    <div class="percentage"><h2 %=@profile.keys[11]%></h2><a href="https://www.codecademy.com/learn/ruby">Ruby Track</a></div>      
+    <div class="percentage"> <h2 %=@profile.keys[13]%></h2><a href="https://www.codecademy.com/learn/web">Web Track</a></div>
+    <br>  
+    <form action="/update" method="post">
+        If you don't see Codecademy profile name, please, type it here: <input type="text" name="trainee[codecademy]" value="i.e. IrynaBi"/>
+        <input type="submit" value="Submit"/>
+    <!--<% @profile.each do |field, value| %>
     <div class="profile-section box">
       <h4><%= field %></h4>
       <p class="profile-value"><%= value %></p>
     </div>
-  <% end %>
+    <% end %> -->
 
-  <div id="stats" class="box">
+    <div id="stats" class="box">
     <h3>Stats</h3>
-    <!-- TODO - Set the contents of this span with JavaScript -->
-    <span id="word-count">0</span> words found.
-  </div>
+    <!-- TODO - Set the contents of this span with JavaScript 
+    <span id="word-count">0</span> words found. -->
+
+    <!-- I wanted to do STAT in more interactive way, 
+    but it shows only 10 words what is definitely not correct. 
+    I keep working on it.-->
+        <script type="text/javascript">
+            function countWords() {
+            profile_values = document.getElementsByClassName('profile-value');
+            total_words = 0;
+            for (i = 0; i < profile_values.length; i++) {
+                total_words += profile_values[i].innerHTML.split(' ').length;
```

Comment:
Sometimes it happens that a bit of whitespace creeps in between HTML elements that might confuse your word count... Try looking at `profile_values[0].innerHTML.split(' ')` and see if you can spot any whitespace. If so see what the `trim` method does and whether that addresses the problem...

https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/String/split